### ISSUE TITLE
Shop Bug

### ISSUE THREAD

**Issue by ivandelp** [SirLeezus/OneStopShop#7]
If I add an item that I have in other menu, that price it's the same on all menus. In my case I add the item to highest price than the other menu I have that item.

**Comment by LeeTheTech** [comment]
Yeah, it's designed like that. In what case would you want exactly the same item in two different shops with different prices? Keep in mind it compares itemstacks, not just the material.

**Comment by ivandelp** [comment]
For example If I add the same item in other menu for weekends by discount. Player can buy the item by normal price or If he remember the new menu added, he buy the same item by discount. Player can take advantage of this offer only weekends.

Why in separated menu?

To have there all items by discount and easy to center that items there or modify more easy.

**Comment by LeeTheTech** [comment]
I think in this case it would be better as a new feature where the current items get adjusted. Redesigning this to work that way would require an entire recode just about. That's a large task for me so I'm not sure when I'll be able to get to it.

**Comment by ivandelp** [comment]
Okey no problem
